source/posts/2015-06-05-getting-started-with-ios-development.markdown

```diff
@@ -0,0 +1,102 @@
+
+That's not to say that having Objective-C knowledge isn't valuable. It's
+especially important to have Objective-C experience to work with older
+applications. For example, [Tropos](http://troposweather.com/) is built with
+Objective-C. In order to contribute to this project, I need to think back to my
+Objective-C learnings. The specifics around iOS development tend to be the
+frameworks that Apple provides to its developers. These frameworks have great
+documentation and are documented in both, Swift and Objective-C. If you're
+learning how to use these libraries and the concepts behind iOS, it shouldn't be
+difficult to apply them to Objective-C.
+
+I was disappointed to see that there weren't many resources to get started in
+iOS from a beginner's perspective with Swift. After spending some more time
+searching I found that Stanford University has a free course on iTunes
+university titled [Developing iOS 8 Apps with
+Swift](https://itunes.apple.com/us/course/developing-ios-8-apps-swift/id961180099).
+This has been an immensely useful resource. Paul Hegarty is a great instructor
+and is really good at explaining the concepts and demonstrating things in real
+time. I highly recommend checking it out.
+
+These are a few of the important things I've learned so far:
+
+* **[Read the iOS Human Interface
+  Guidelines](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/)**
+  This may go without saying but Apple has done a ton of research behind the
+  user experience of iOS and its native applications. Fun fact that I learned:
+  The loading screen for an iOS application shouldn't be used as a splash page.
+  Instead, it should be a representation of the inactive application to give the
+  illusion of a quick load.
+
+* **Have a project idea** While reading and following the Stanford online course
+  was immensely useful to start learning, I always feel that I need a concrete
+  project to work on to apply my learnings. Without this, I usually can't keep
+  sustained interest. I recommend coming up with a simple app idea to work on to
+  build while learning. I especially got tired of the calculator application
+  example from the Stanford course.
+
+* **Read through Apple's developer documentation** While developing for iOS you
+  have a huge amount of frameworks at your disposal provided to you by Apple.
+  Apple also provides excellent documentation around all of these frameworks.
+  XCode even has a documentation browser built in. Familiarize yourself with
+  these documents and learn to navigate them.
+
+* **Use the Apple developer forums** I wasn't immediately aware of how useful
+  the Apple developer forums would be. I was running into trouble getting an
+  archive of my app sent to iTunes connect and spent hours if not days searching
+  for support. Most of my searching turned up various StackOverflow posts which
+  were not helpful. After checking in the thoughtbot XCode slack channel I found
```

Comment:
change: Most of my searching turned up various StackOverflow posts which were not helpful. After checking in the thoughtbot XCode slack channel I found..."

update to: "Most of my searching turned up various StackOverflow posts which were not helpful but, after checking in the thoughtbot XCode slack channel I found that someone else ..."